packages/client/src/generation/generateClient.ts

```diff
@@ -135,6 +135,25 @@ export async function buildClient({
     fileMap['edge.d.ts'] = await TS(edgeTsClient, true)
   }
 
+  if (generator?.previewFeatures.includes('denoDeploy') && !!globalThis.Deno) {
+    // we create a client that is fit for edge runtimes
+    const denoTsClient = new TSClient({
+      ...tsClientOptions,
+      dataProxy: true, // edge only works w/ data proxy
+      runtimeName: 'index.d.ts',
+      runtimeDir: '../' + runtimeDirs.edge,
+      denoDeploy: true,
+    })
+
+    fileMap['deno/edge.js'] = await JS(denoTsClient, true)
+    fileMap['deno/index.d.ts'] = await TS(denoTsClient)
+    fileMap['deno/edge.ts'] = `import 'https://deno.land/std@0.152.0/dotenv/load.ts'
```

Comment:
I this is loading `.env` files, then this is not needed. Edge clients don't support this:
https://www.prisma.io/docs/data-platform/data-proxy#edge-runtimes